src/main/pages/che-7/administration-guide/proc_authenticating-to-the-che-server-using-openid.adoc

```diff
@@ -50,7 +50,7 @@ If the validation is successful, the parsed form of the token is passed to the e
 
 . Environment initialization: The filter extracts data from the JWT token claims, creates the user in the local database if it is not yet present, and constructs the `subject` object and sets it into the per-request *EnvironmentContext* object, which is statically accessible everywhere.
 +
-If the request was made using only a machine token, the following single authentication filter is used:
+If the request was made using only a JWT token, the following single authentication filter is used:
```

Comment:
this might be incorrect now. not all JWT tokens would be handled by MachineLoginFilter